ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -1066,10 +1071,14 @@ def orthogonal_polynomial(
     "--poly_order", default="first", help="can do first and second order so far"
 )
 @click.option(
-    "--precomputed", default=False, help="if true, then saved results are used"
+    "--precomputed",
+    type=click.Path(),
+    help="directory which contains results from a previous run",
 )
 @click.option(
-    "--out_dir", help="directory to save output/debug files to", type=str
+    "--out_dir",
+    help="directory to save output/debug files to",
+    type=str,
```

Comment:
Might be good to change the type for `out_dir` to `click.Path` at some point.

Reply:
Yes, good idea!